README.md

```diff
@@ -71,30 +71,73 @@ make install
-following build options are needed to be set when run bjam.exe: msvc architecture=x86 address-model=64 link=static runtime-link=static stage
-all lib will be generated except boost_zlib:
-download [zlib source](http://gnuwin32.sourceforge.net/downlinks/zlib-src-zip.php "zlib source") and extract to directory C:/zlib
-run cmd:bjam.exe msvc architecture=x86 address-model=64 link=static runtime-link=static --with-iostreams -s ZLIB_SOURCE=C:\zlib\src\zlib\1.2.3\zlib-1.2.3 stage
+##### 1. install [libevent 2.0.22](https://github.com/libevent/libevent/archive/release-2.0.22-stable.zip "libevent 2.0.22")
+Extract libevent to LocalPath
+copy files in [libevent-2.0.22-stable\WIN32-Code\] to [libevent-2.0.22-stable\include\]
+Open Visual Studio command line tools(x64)
+```shell
+cd libevent-2.0.22-stable
+nmake /f Makefile.nmake
+dir *.lib
+mkdir lib
+copy libevent.lib lib\
+copy libevent_core.lib lib\
+copy libevent_extras.lib lib\
+```
+##### 2. install [jsoncpp 0.10.6](https://github.com/open-source-parsers/jsoncpp/archive/0.10.6.zip "jsoncpp 0.10.6")
+Extract jsoncpp to LocalPath
+```shell
+1.Open Visual Studio with [jsoncpp-0.10.6\makefiles\msvc2010\jsoncpp.sln]
+  Then perheps there is a message box popup, click OK.
+2.Set [Configuration]:Release at Toolbar
+  Set [Platform]:x64 at Toolbar
+3.Right click project [lib_json], and click [build], 
+  and then Visual Studio will compile the lib file at [x64/Release]
+4.copy [jsoncpp-0.10.6\makefiles\msvc2010\x64\Release\lib_json.lib]
+  to [jsoncpp-0.10.6\lib\lib_json.lib]
+```
+##### 3. install [boost 1.56.0](http://sourceforge.net/projects/boost/files/boost/1.56.0/boost_1_56_0.zip "boost 1.56.0") and [zlib 1.2.3](http://gnuwin32.sourceforge.net/downlinks/zlib-src-zip.php "zlib 1.2.3")
+Extract [boost 1.56.0] and [zlib] to LocalPath
+modify "#if 1" to "#if HAVE_UNISTD_H" at [zlib-1.2.3-src\src\zlib\1.2.3\zlib-1.2.3\zconf.h]
+Open Visual Studio command line tools(x64)
+```shell
+cd boost_1_56_0
+booststrap.bat
+bjam.exe --toolset=msvc-12.0 architecture=x86 address-model=64 link=static runtime-link=static stage --stagedir="[Your Loacl Path]\boost_1_56_0\lib" threading=multi variant=release --build-type=complete --with-iostreams -s ZLIB_SOURCE=[Your Loacl Path]\zlib-1.2.3-src\src\zlib\1.2.3\zlib-1.2.3
+```
+[Your Loacl Path] is the directory that you extract the zip file. 
 
 #### Make and Install
-run cmake-gui.exe, choose your source code dir and build dir, then click generate which will let you choose VirtualStudio version
-if generate project solution fails, change BOOST_INCLUDEDIR/LIBEVENT_INCLUDE_DIR/JSONCPP_INCLUDE_DIR in CMakeList.txt, according to its real install path
-open&build&run project by VirtualStudio
+##### 1.get source
+```shell
+git clone https://github.com/apache/rocketmq-client-cpp.git
+```
+##### 2. configure path
+open the CMakeLists.txt at [rocketmq-client-cpp] and modify the config.

```

Comment:
It is necessary to specify which configuration items need to be modified and which configuration items need to be newly added.

Reply:
Ok, we update the CMakeList.txt of Linux at first. windows config will be modified when the linux version confirmed.